wom/src/main/scala/wdl4s/wdl/WdlGraphNode.scala

```diff
@@ -40,6 +46,24 @@ object WdlGraphNode {
     val updatesNeeded = graphNode.upstream -- currentSet
     updatesNeeded.foldLeft(setWithUpstream)(calculateUpstreamAncestry)
   }
+
+  private[wdl] def outputPortFromNode(node: WdlGraphNode, terminal: Option[Terminal]): ErrorOr[GraphNodePort.OutputPort] = {
+
+    def findNamedOutputPort(name: String, graphOutputPorts: Set[GraphNodePort.OutputPort], terminalName: String): ErrorOr[GraphNodePort.OutputPort] = {
+      graphOutputPorts.find(_.name == name).toValidNel(s"Cannot find an output port $name in $terminalName")
+    }
```

Comment:
They give me a warm and fuzzy sense of security, so I tend to add them even when not strictly required when splitting a one-liner over two lines.